### PR TITLE
fix: [CDS-38231]: integrated repowizard with checkbox

### PR DESCRIPTION
**Pull request by sahithikolichala**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx [hunk 1]
```diff
@@ -112,19 +113,34 @@ export default function ManifestSelection({
...
-        stage={stage}
-        connectors={fetchedConnectorResponse}
-        refetchConnectors={refetchConnectorList}
-        listOfManifests={listOfManifests}
-        isReadonly={readonly}
-        deploymentType={deploymentType}
-        allowableTypes={allowableTypes}
-        allowOnlyOne={isServerlessDeploymentType(deploymentType)}
-      />
+      {!(pipeline as any).gitOpsEnabled ? (
```
Comment (bhavya-sinha) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx:116]: Use proper type

File: src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/DeployServiceDefinition/DeployServiceDefinition.tsx [hunk 2]
```diff
@@ -165,15 +166,16 @@ function DeployServiceDefinition(): React.ReactElement {
...
-          disabled={true}
-        />
+        <Card className={stageCss.sectionCard}>
+          <Checkbox
+            label="Gitops"
+            name="gitOpsEnabled"
+            checked={gitOpsEnabled}
+            onChange={handleGitOpsCheckChanged}
+            disabled={disabledState}
+            //GitOps checkbox is disabled temporarily until Release repo manifest is added
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/DeployServiceDefinition/DeployServiceDefinition.tsx:176]: You can remove this now

**Review by bhavya-sinha — CHANGES_REQUESTED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
retrigger jest

File: src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx [hunk 3]
```diff
@@ -99,11 +107,14 @@ export default function ManifestSelection({
...
       const connectorIdentifiers = connectorList.map((item: { scope: string; identifier: string }) => item.identifier)
       const response = await fetchConnectors({ filterType: 'Connector', connectorIdentifiers })
-      if (response?.data) {
+      /* istanbul ignore else */
+      if (get(response, 'data', null)) {
         const { data: connectorResponse } = response
         setFetchedConnectorResponse(connectorResponse)
       }
     } catch (e) {
+      /* istanbul ignore next */
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx:116]: Do we need both?

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx (hunk 3 above)
Comment (AshwinBhatkal-Harness) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestSelection.tsx:116]: You can remove this in your PR. LGTM

**Pull request merged**